modules/rxrtbBidAdapter.js

```diff
@@ -100,7 +100,7 @@ function makeImp(req) {
     'banner': makeBanner(req)
   };
 
-  if (req.params.bidfloor && utils.isInteger(req.params.bidfloor)) {
+  if (req.params.bidfloor && utils.isFinite(req.params.bidfloor)) {
```

Comment:
The `utils` module doesn't expose an `isFinite` function, but `isFinite` is available on the global object directly so if you remove `utils.` this should work